src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ImportController.php

```diff
@@ -86,6 +87,13 @@ public function importAction(Request $request)
         $iniConfiguration = $this->get('prestashop.core.configuration.ini_configuration');
 
         $form = $formHandler->getForm();
+
+        if ($request->query->has('import_type')) {
+            $formData = $form->getData();
+            $formData['entity'] = Entity::getFromName($request->query->get('import_type'));
```

Comment:
You can use form transformer instead? More, not sure @mickaelandrieu will be agree to add logic here :(